src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplate.java

```diff
@@ -381,7 +381,7 @@ public String getActiveDeadlineSecondsStr() {
         ImmutableMap.Builder<String, String> builder = ImmutableMap.<String, String> builder();
         if (!labelSet.isEmpty()) {
             for (LabelAtom label : labelSet) {
-                builder.put(label == null ? DEFAULT_ID : "jenkins/" + label.getName(), "true");
+                builder.put("buildlabel",label == null ? DEFAULT_ID : "jenkins_" + label.getName());
```

Comment:
Why not `jenkins/label` as a key?

Reply:
No reason it can't be that - will change it.